package-examples/ghost/ghost/secret-example-ghost.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
This is ok for the initial version, but we don't want actual Secret resources in packages, especially not in git. We'll want a configuration that specifies how to generate or access the Secret instead. See #3125

Reply:
We discussed and decided to treat secret as an external resources (not managed by our stack for now). So secrets will not be part of the package.